package.json

```diff
@@ -71,6 +72,7 @@
     "fs-extra": "^5.0.0",
     "google-libphonenumber": "^3.0.7",
     "got": "^8.2.0",
+    "linkify-it": "^2.0.3",
```

Comment:
You mentioned something about a fork? Should this be a different dependency?

Reply:
As discussed, I copied `link-test` into our repo and `linkify-it` is the underlying module that parses links.